pkg/installer/utils.go

```diff
@@ -214,3 +218,102 @@ func pullManifest(url string) (string, error) {
 	}
 	return string(contents), nil
 }
+
+// storageClassToManifest returns a manifest for storageClass
+func storageClassToManifest(storageClass *kstoragev1.StorageClass) ([]byte, error) {
+	tmp := &kstoragev1.StorageClass{}
+	tmp.APIVersion = "storage.k8s.io/v1"
+	tmp.Kind = "StorageClass"
+	tmp.Name = storageClass.Name
+	tmp.Namespace = storageClass.Namespace
+	tmp.Provisioner = storageClass.Provisioner
+	tmp.Parameters = storageClass.Parameters
+	tmp.ReclaimPolicy = storageClass.ReclaimPolicy
+	tmp.MountOptions = storageClass.MountOptions
+	tmp.AllowVolumeExpansion = storageClass.AllowVolumeExpansion
+	tmp.VolumeBindingMode = storageClass.VolumeBindingMode
+	tmp.AllowedTopologies = storageClass.AllowedTopologies
+
+	data, err := json.Marshal(&tmp)
+	if err != nil {
+		return nil, err
+	}
+	data, err = gyaml.JSONToYAML(data)
+	if err != nil {
+		return nil, err
+	}
+	return data, nil
+}
+
+// storageClassToManifest returns a manifest for storageOSCluster
+func storageOSClusterToManifest(storageOSCluster *operatorapi.StorageOSCluster) ([]byte, error) {
+	tmp := &operatorapi.StorageOSCluster{}
+	tmp.APIVersion = storageOSCluster.APIVersion
+	tmp.Kind = storageOSCluster.Kind
+	tmp.Name = storageOSCluster.Name
+	tmp.Namespace = storageOSCluster.Namespace
+	tmp.Spec = storageOSCluster.Spec
+
+	data, err := json.Marshal(&tmp)
+	if err != nil {
+		return nil, err
+	}
+	data, err = gyaml.JSONToYAML(data)
+	if err != nil {
+		return nil, err
+	}
+	return data, nil
+}
+
+// secretToManifest returns a manifest for secret
+func secretToManifest(secret *corev1.Secret) ([]byte, error) {
+	tmp := &corev1.Secret{}
```

Comment:
Why we need to highlight with the name that this is a temporary secret?

Reply:
same as above, how about `newSecret`?

Reply:
yepp, thanks